src/components/About/About.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import img from '../images/nike.jpg';
+import Box from '@material-ui/core/Box';
+import { Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Styles should be in a separate .scss file which is imported from javascript file.